frontend/src/stimulus/controllers/password-confirmation-dialog.controller.ts

```diff
@@ -68,24 +68,27 @@ export default class PasswordConfirmationDialogController extends ApplicationCon
     }
 
     this.activeDialog = true;
-    opModalService
-      .show(PasswordConfirmationModalComponent, 'global')
-      .subscribe((modal) => modal.closingEvent.subscribe(() => {
-        this.activeDialog = false;
-
-        if (!modal.confirmed) {
-          return;
-        }
-
-        const input = document.createElement('input');
-        input.type = 'hidden';
-        input.id = 'hidden_password_confirmation';
-        input.name = '_password_confirmation';
-        input.value = modal.password_confirmation as string;
-
-        form.append(input);
-        form.requestSubmit(event?.submitter);
-      }));
+
+    pluginContext.runInZone(() => {
```

Comment:
Since I am not really understanding what's happening, I can at least question the diff I observe. In the other file we call `runInZone` on a context that we get from `pluginContext.subscribe`.  Why is this indirection not necessary here?